src/__screenshot_tests__/counter-screenshot-test.tsx

```diff
@@ -0,0 +1,75 @@
+import {openStoryPage, screen} from '../test-utils';
+
+const SKINS = ['Movistar', 'O2', 'Vivo', 'Vivo-new', 'Telefonica', 'Blau'] as const;
+const INVERSE_VALUES = [false, true];
+const DARK_MODE_VALUES = [false, true];
+
+const getCases = () => {
+    const cases = [];
+    for (const skin of SKINS) {
+        for (const isInverse of INVERSE_VALUES) {
+            for (const isDarkMode of DARK_MODE_VALUES) {
+                cases.push([skin, isInverse, isDarkMode]);
+            }
+        }
+    }
+
+    return cases;
+};
+
+test.each(getCases())(
+    'Counter default. skin={%s} isInverse={%s} isDarkMode={%s}',
+    async (skin, isInverse, isDarkMode) => {
+        await openStoryPage({
+            id: 'components-counter--default',
+            device: 'DESKTOP',
+            args: {inverse: isInverse, defaultValue: 1},
+            isDarkMode: isDarkMode as boolean,
+            skin: skin as (typeof SKINS)[number],
+        });
+
+        const counter = await screen.findByTestId('counter');
+
+        const image = await counter.screenshot();
+        expect(image).toMatchImageSnapshot();
+    }
+);
+
+test.each`
+    min  | max   | defaultValue | removable | disabled
```

Comment:
does it make sense to test all these combinations?

Reply:
Yes, these are different scenarios for when some of the controls are disabled because of value being equal to min/max, or when the buttons are disabled but the counter itself had a disabled state.

I've removed many useless tests that were combining different skins with inverse/darkMode